### PR TITLE
fixed onlinelibraryfragment

### LINKED ISSUE #2339: Merge LibraryFragment with OnlineLibraryFragment
**Is your feature request related to a problem? Please describe.**

`LibraryFragment` is only inherited by one class: `OnlineLibraryFragment`. By merging these two classes we reduce duplicated code and number of files.

### PR DESCRIPTION
**Pull request by MaskedCarrot**
Fixes #2339

**Screenshots**

### CODE REVIEW

File: app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt [hunk 1]
```diff
@@ -28,29 +28,38 @@ import android.view.ViewGroup
...
+import org.kiwix.kiwixmobile.core.extensions.ActivityExtensions.viewModel
 import org.kiwix.kiwixmobile.core.main.CoreMainActivity
 import org.kiwix.kiwixmobile.core.utils.SimpleTextListener
 import org.kiwix.kiwixmobile.language.LanguageActivity
-import org.kiwix.kiwixmobile.zim_manager.library_view.LibraryFragment
+import org.kiwix.kiwixmobile.zim_manager.ZimManageViewModel
+import javax.inject.Inject
 
-class OnlineLibraryFragment : LibraryFragment(), FragmentActivityExtensions {
+class OnlineLibraryFragment : BaseFragment(), FragmentActivityExtensions {
```
Comment (macgills) [inline app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt:45]: I don't think this is what was intended from the ticket.

The idea was there shouldn't be 2 classes when there is only 1 actually used type. LibraryFragment & OnlineLibraryFragment should become one class

**Comment by macgills** [comment]
@MaskedCarrot will you be able to devote the time to complete this PR?

It is our policy to close PRs once they have 2 successive stale markings

**Comment by MaskedCarrot** [comment]
I am sorry I won't be able to work on this right now

**Comment by macgills** [comment]
Perfectly alright, closing. Hope to see contributions from yourself in the future

File: app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt (hunk 1 above)
Comment (s-ayush2903) [reply app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt:45]: So ain't we going to delete `LibraryFragment` and put its required content appropriately in `OnlineLibFragment`?

Comment (macgills) [reply app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt:45]: We are going to do it, this contributor doesn't have the time to

**Pull request closed without merging**